src/imitation/scripts/config/common.py

```diff
@@ -1,7 +1,10 @@
 DEFAULT_BLANK_POLICY_KWARGS = dict(
+    # For recommended PPO hyperparams in each environment, see:
+    # https://github.com/araffin/rl-baselines-zoo/blob/master/hyperparams/ppo2.yml
     learning_rate=3e-4,
     nminibatches=32,
     noptepochs=10,
-    # WARNING: this is actually 8*256=2048 due to 8 vector environments
-    n_steps=256,
+    # WARNING: this is actually 8*2048=16384 due to 8 vector environments
+    # (n_steps=2048 across 8 vector environments is also what the Zoo uses)
+    n_steps=2048,
```

Comment:
Bigger `n_steps` means lower variance gradients but fewer overall gradient steps. So it should always work but might require `total_timesteps` be increased.

I ended up using a batch size of 16,384 / n_steps=2048 for Ant and Humanoid and it worked much better in those environments than n_steps=256. For things like CartPole and even Pendulum I expect a smaller n_steps will work fine and be faster overall. For things like HalfCheetah it could go either way, would need to be benchmarked.

Reply:
Perhaps it's worth setting up the default config so that it's environment-dependent? e.g. if someone runs on Pendulum then it should use this value, if someone runs on CartPole it should use 32, etc. I could add such a change to this PR or include it in a new one.

Reply:
I like having sensible defaults, but making the hyperparams change based on `env_name` feels too magical. How about just a named config for each environment, setting the `env_name` and appropriate hyperparams?